Message.py

```diff
@@ -0,0 +1,17 @@
+class Message:
+    """
+    A messages class.
+    Users send messages using this class.
+    """
+
+    def __init__(self, message_id, body, sender):
+        self.read = False
+        self.sender = sender
+        self.body = body
+        self.id = message_id
+
+    def __str__(self):
+        return "{sender} send the message: {message}".format(sender=self.sender,message=self.body)
+
+    def __repr__(self):
+        return self.__str__()
```

Comment:
What you are you doing here? __str__ and __repr__ may both return string but have different uses.
https://www.journaldev.com/22460/python-str-repr-functions